### PR TITLE
Switch to js

### PR DESCRIPTION
**Pull request by butera-simone**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  await mkdir('platform-build/bin/pear-runtime-app', { recursive: true })
+  await mkdir('platform-build/lib', { recursive: true })
+
+  await rename('vendor/electron-runtime/dist/linux-unpacked', 'platform-build/bin/pear-runtime-app')
+  await rename('vendor/wakeup/build/bin/pear', 'platform-build/bin/pear')
+  await rename('vendor/pear-runtime-bare/build/pear-runtime', 'platform-build/bin/pear-runtime')
+  await rename('vendor/libappling/build/launch.so', 'platform-build/lib/launch.so')
+}
+
+main()
```
Comment (mafintosh) [inline index.js]: you can just make this esm, then you dont need the main wrapper. type: 'module' in pkg json

Comment (butera-simone) [reply index.js]: done

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+platform-build
```
Comment (mafintosh) [inline .gitignore:3]: lets just call this build now as its in the repo

**Comment by mafintosh** [comment]
@butera-simone fix the above comment in another PR

**Pull request merged**